nearcore/src/config.rs

```diff
@@ -36,6 +37,45 @@ use near_primitives::version::PROTOCOL_VERSION;
 use near_rosetta_rpc::RosettaRpcConfig;
 use near_telemetry::TelemetryConfig;
 
+/// `blacklist_from_iter` - converts list of addresses represented by strings to <IpAddr, BlockedPorts> HashMap
+///
+/// Arguments:
+/// - `blacklist`- list of strings in following formats:
+///    - "IP" - for example 127.0.0.1 - if only IP is provided we will block all ports
+///    - "IP:PORT - for example 127.0.0.1:2134
+pub fn blacklist_from_iter<T>(blacklist: T) -> HashMap<IpAddr, BlockedPorts>
+where
```

Comment:
I am not sure whether it makes sense for this function to be in this file. It feels like a function specific to network

Reply:
@bowenwang1996 It doesn't. I removed it out and moved it out to another package in another PR.

Reply:
@bowenwang1996 This function is not used in network at all. It's used only for parsing config. Though we can put it inside near-network-primitives if you think that's better.

Reply:
yeah that might be better